BLUESPAWN-win-client/src/hunt/hunts/HuntT1546.cpp

```diff
@@ -41,6 +43,27 @@ namespace Hunts {
         files.emplace(file, std::vector<RegistryValue>{ __VA_ARGS__ }); \
     }
 
+    void HuntT1546::Subtechnique002(IN CONST Scope& scope, OUT std::vector<std::shared_ptr<Detection>>& detections){
+        SUBTECHNIQUE_INIT(002, Screensaver);
+
+        SUBSECTION_INIT(SCREENSAVER_KEY, Cursory);
+        for(const auto& detection : CheckValues(HKEY_CURRENT_USER, L"Control Panel\\Desktop",
+                                                { { L"SCRNSAVE.exe", L"", false, CheckSzEmpty } })){
+            CREATE_DETECTION(Certainty::None,
+                             RegistryDetectionData{ detection, RegistryDetectionType::CommandReference });
+        }
+        SUBSECTION_END();
+
+        SUBSECTION_INIT(SCREENSAVER_FILE, Cursory);
+        auto path{ L"C:\\Windows\\System32\\scrnsave.scr" };
+        if(FileSystem::CheckFileExists(path) && !FileSystem::File{ path }.GetFileSigned()){
```

Comment:
scrnsave.scr isn't signed by default, though?

Reply:
you could maybe check that it's owned by TrustedInstaller

Reply:
I didn't get anything when I tested

Reply:
ah I just ran sigcheck against the one on my system and it does show as signed. Must be a cabinet which is why it didn't show in explorer